pkg/proc/gdbserial/gdbserver.go

```diff
@@ -1327,6 +1327,9 @@ func (p *Process) loadGInstr() []byte {
 	case "darwin":
 		// mov rcx,QWORD PTR gs:{uint32(off)}
 		op = []byte{0x65, 0x48, 0x8B, 0x0C, 0x25}
+	case "freebsd":
```

Comment:
Let's clean this up a bit by just merging this into the case above, e.g. `case "darwin", "freebsd":`